### PR TITLE
WIP: Add invoke tasks for unit tests and linter

### PR DESCRIPTION
**Pull request by markdgray**
This PR adds tasks to "invoke" in order to enable convenient execution of the linter and unit-test stage from circleci.

### CODE REVIEW

**Comment by russellb** [comment]
partial duplicate of #707, but I didn't have a target for the linting.  Maybe rebase on #707?

**Comment by markdgray** [comment]
> partial duplicate of #707, but I didn't have a target for the linting. Maybe rebase on #707?

I didnt see this. I will close this one. Thanks @russellb

**Comment by russellb** [comment]
@markdgray I merged #707.  If you want to rebase, you could re-open this for the lint part.  That wasn't included in the other PR.

File: tasks.py [hunk 1]
```diff
@@ -444,3 +444,16 @@ def _replace(pattern):
...
+def unit_tests(ctx):
+    """Run unit tests."""
+    run("go test ./...")
+    run("go test -race ./...")
+
+@task
+def linter(ctx):
+    """Run linter."""
+    # Install "gometalinter" and run linter
+    run("curl -L https://git.io/vp6lP | sh")
```
Comment (russellb) [inline tasks.py:458]: Is there a container we could run that includes this?  It's one thing to do this automatically in the CI job, but some people might be surprised that we install stuff on their dev machine automatically like this.

Comment (markdgray) [reply tasks.py:458]: Possibly. I looked at your PR, and like you I was just copying the CI build instructions. I am happy to investigate some improvements like you suggest above but it looks like your PR is near to merge. Perhaps I should wait for merge and then build on top of it.

**Pull request closed without merging**